### PR TITLE
[CHORE] upgrading otel version

### PR DESCRIPTION
**Pull request by nicolastakashi**
I'm upgrading the OTEL version to match the same config we have on this [PR](https://github.com/coralogix/telemetry-shippers/commit/9ebd56ae3bd6705879f863ca70dad892bbef5f9c).

### CODE REVIEW

**Review by ShauliSolomovich — APPROVED** [review]

File: otel-infrastructure-collector/k8s-helm/values.yaml [hunk 1]
```diff
@@ -14,10 +15,10 @@ opentelemetry-collector:
...
     name: "otel-infrastructure-collector"
     create: true
-    rules: 
-    - apiGroups: ["", "events.k8s.io"]
-      resources: ["events"]
-      verbs: ["watch", "list"]
+    rules:
+      - apiGroups: ["", "events.k8s.io"]
+        resources: ["events"]
+        verbs: ["watch", "list"]
```
Comment (matej-g) [inline otel-infrastructure-collector/k8s-helm/values.yaml:21]: These indentation changes across PR are intended?

**Review by matej-g — APPROVED** [review]
Looks good 👍 One question

File: otel-infrastructure-collector/k8s-helm/values.yaml (hunk 1 above)
Comment (nicolastakashi) [reply otel-infrastructure-collector/k8s-helm/values.yaml:21]: The lint runs to format the yaml file, I can revert if needed 😄

Comment (matej-g) [reply otel-infrastructure-collector/k8s-helm/values.yaml:21]: All good, I'm just always suspicion of indentation changes in YAMLs 😉, better linted and not linted!

**Pull request merged**